documentation/sphinx/source/release-notes.rst

```diff
@@ -45,6 +45,7 @@ Other Changes
 -------------
 
 * Trace files are now ordered lexicographically. This means that the filename format for trace files did change. `(PR #1828) <https://github.com/apple/foundationdb/pull/1828>`_.
+* ``MakoWorkload``, used as a benchmark to do performance test of FDB. `(PR #1586) <https://github.com/apple/foundationdb/pull/1586>`_.
```

Comment:
Generally the release notes items would be formatted with a verb, such as:

```suggestion
* `Added `MakoWorkload``, used as a benchmark to do performance testing of FDB. `(PR #1586) <https://github.com/apple/foundationdb/pull/1586>`_.
```

Reply:
Done. Thanks.